Modules/signalmodule.c

```diff
@@ -423,7 +423,7 @@ signal_raise_signal_impl(PyObject *module, int signalnum)
     err = raise(signalnum);
     _Py_END_SUPPRESS_IPH
     Py_END_ALLOW_THREADS
-    
+
```

Comment:
Please don't commit whitespace and cosmetic changes.

Reply:
My bad. I have had previous formatting issues whenever I make changes to code on another project. i think my editor messes sometimes. I will look into the diff and remove these.